Sources/Helpers/OAAppSettings.m

```diff
@@ -1266,6 +1266,115 @@ - (NSString *)description
 
 @end
 
+@interface OADownloadMode()
```

Comment:
Extract to a separate file. Why not use the static getters like in other Java-like enums that we turned into classes?